### PR TITLE
Add the ability to skip sleeps.

### PR DESCRIPTION
**Pull request by robholland**
This makes workflow end to end latency more informative by avoiding dilution with un-optimisable time.

### CODE REVIEW

File: workers/go/throughputstress/workflow.go
Comment (cretz) [inline workers/go/throughputstress/workflow.go:39]: I wonder if instead of making an `UpdateSleep` update stop sleeping, you should avoid calling `UpdateSleep` altogether if this flag is set.

Comment (robholland) [reply workers/go/throughputstress/workflow.go]: Yeah, makes sense. Adjusted to do that.

**Review by cretz — APPROVED** [review]
Works for me. But if possible, can we wait for @Sushisource's review before merging?

File: workers/go/throughputstress/workflow.go
Comment (robholland) [reply workers/go/throughputstress/workflow.go:144]: heh, @cretz suggested the opposite, so I moved it here. I prefer it this way, but thanks for checking it :)

Comment (robholland) [reply workers/go/throughputstress/workflow.go:144]: As we have other Update handlers that will still kick in, I'll leave it this way.

**Pull request merged**
